src/plots/cartesian/layout_attributes.js

```diff
@@ -698,6 +698,22 @@ module.exports = {
             'In other cases the default is *hide past div*.'
         ].join(' ')
     },
+    ticklabelshiftx: {
```

Comment:
Wondering where we should put the `x` or `y` in the attribute names?
In #7005 we have `xshift`, here we have `shiftx`.

Reply:
Hm, true. in #7005 my reasoning was that in the shape reference we have x0, x1 and xref. So I added xshift.

Here I had a similar reasoning, there were already properties starting with ticklabel*. And since in `ticklabelxshift` it is maybe hard to make out the 'x', I moved it to the end…

No strong opinion here.

Reply:
IMHo we shouldn't use `x` and `y` in these attribute names.
Instead something like `standoff` and `runoff` could be used so that when switching the orientation e.g. on a colorbar from horizontal to vertical everything works without a need to adjusting these parameters.